.github/workflows/ci.yml

```diff
@@ -29,6 +29,10 @@ jobs:
       - name: Set TRUFFLERUBYOPT
         run: echo "TRUFFLERUBYOPT=--jvm --polyglot" >> $GITHUB_ENV
         if: startsWith(matrix.ruby, 'truffleruby+graalvm')
+      - name: Install GraalVM js component
+        run: gu install js
```

Comment:
Is there a way we could do this with a version check or something? This way we merge it now, and when 22.2 is out it keeps working.